CONTRIBUTING.md

```diff
@@ -1,195 +1,137 @@
 
-Before you submit your first contribution please accept our Contributor License Agreement (CLA) here. When you send your first Pull Request (PR), GitHub will ask you to accept the CLA.
+    >**Tip:** If you are unsure whether you have found a bug, search existing issues in the corresponding repo on GitHub and raise it in the [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev).
+    - Compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo
+    - Language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo
+    - Website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo
+    - Security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
 
-There is no need to do this before you send your first PR.
+-  Start with GitHub issues that can be fixed easily:
+    - Browse issues labeled "good first issue" in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+    - Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
 
-Subsequent PRs will not require CLA acceptance.
+## Set up the prerequisites
+1. Download [Ballerina](https://ballerina.io) and go through the [learning resources](https://ballerina.io/learn).
+2. Read the <a href="/code-of-conduct">Ballerina Code of Conduct</a>.
 
-If for some (unlikely) reason at any time CLA changes, you will get presented with the new CLA text on your first PR after the change.
+## Build the source code 
 
-### Obtaining the Source Code and Building the Project
+For instructions, see <a href="/learn/installing-ballerina/#installing-from-source">Installing from source</a>.
 
-#### Build Prerequisites
+## Set up the development environment
 
-**Prerequisite 1** - Install a Java Development Kit (JDK) version 8
+See the below sections to set up the required plugins in your preferred IDEs/Editors.
 
-Building Ballerina requires a Java SE Development Kit (JDK) version 8 to be installed. You can download one from one of the following locations:
+### Set up the Ballerina plugin
 
-* [http://www.oracle.com/technetwork/java/javase/downloads/index.html](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* [http://openjdk.java.net/install/index.html](http://openjdk.java.net/install/index.html)
+Currently, Ballerina has the below plugins developed for IntelliJ IDEA and VS Code. 
+- [Ballerina plugin for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina)
+    
+    For installation instructions, see [Ballerina IntelliJ Plugin Documentation](https://ballerina.io/learn/vscode-plugin).
```

Comment:
Incorrect url?

Reply:
Fixed all 4 URLs of the 2 plugin download locations and Docs.